qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -874,14 +874,14 @@ class TwoQubitBasisDecomposer:
     basis gate.
 
     Args:
-        gate (Gate): Two-qubit gate to be used in the KAK decomposition.
-        basis_fidelity (float): Fidelity to be assumed for applications of KAK Gate. Default 1.0.
-        euler_basis (str): Basis string to be provided to OneQubitEulerDecomposer for 1Q synthesis.
-            Valid options are ['ZYZ', 'ZXZ', 'XYX', 'U', 'U3', 'U1X', 'PSX', 'ZSX', 'RR'].
-        pulse_optimize (None or bool): If True, try to do decomposition which minimizes
+        gate: Two-qubit gate to be used in the KAK decomposition.
+        basis_fidelity: Fidelity to be assumed for applications of KAK Gate. Default 1.0.
+        euler_basis: Basis string to be provided to OneQubitEulerDecomposer for 1Q synthesis.
+            Valid options are ``['ZYZ', 'ZXZ', 'XYX', 'U', 'U3', 'U1X', 'PSX', 'ZSX', 'RR']``.
```

Comment:
I see you ;)

Reply:
You've shamed me into adding the individual backticks: [7656a59](https://github.com/Qiskit/qiskit/pull/11755/commits/7656a59d2f4c17bc24072b668a59667874432170)